DQM/DTMonitorClient/src/DTBlockedROChannelsTest.cc

```diff
@@ -59,8 +59,9 @@ void DTBlockedROChannelsTest::fillChamberMap( DQMStore::IGetter & igetter, const
   // get the RO mapping
   context.get<DTReadOutMappingRcd>().get(mapping);
 
-  // fill the map of the robs per chamber
-  for(int dduId = FEDNumbering::MINDTFEDID; dduId<=FEDNumbering::MAXDTFEDID; ++dduId) { //loop over DDUs
+  // fill the map of the robs per chamber 
+  // //FIXME: monitoring only real used FEDs
+  for(int dduId = FEDNumbering::MINDTFEDID; dduId<=774; ++dduId) { //loop over DDUs
```

Comment:
hi @jfernan2 - this is the wrong way to fix whatever problem you are seeing. MAXDTFEDID is the right range of the loop - your conditions and the data itself know what feds are in use (as that can change during data taking).

is the issue you have summarized somewhere?